### ISSUE TITLE
Feature Request: Support additional SMS gateways

### ISSUE THREAD

**Issue by heyakyra** [ProgressiveCoders/Spoke#1455]
Some people use other SMS/VoIP services such as SignalWire, Plivo, Sinch, or Bandwidth, so it would be ideal to support those too. 

Some basic abilities like this exist in other tools like ERPNext: 
https://github.com/frappe/frappe/pull/9381

**Solution**
The https://github.com/plivo/plivo-node exists so there is an NPM package available.

**Comment by ryanjcrump** [comment]
WFP is very interested in looking at Bandwidth as an option! Anyone know if that work is underway?

**Comment by crayolakat** [comment]
@ryanjcrump Yes! MoveOn has been working on and testing a production-level implementation of Bandwidth. It's scheduled to be released in the 12.2 version of Spoke.
